server/cli/src/backup/restore.rs

```diff
@@ -0,0 +1,197 @@
+use super::*;
+use copy_dir::copy_dir;
+use diesel::{Connection, RunQueryDsl};
+use repository::DBBackendConnection;
+use service::settings::Settings;
+use simple_log::is_debug;
+use std::{fs, io, path::PathBuf, process::Command, str::FromStr};
+
+pub(crate) fn restore(
```

Comment:
thought for later is how we can make the errors look nicer:

```
cargo run --bin remote_server_cli --features postgres -- restore --backup-name D2024_08_26T06_01_27 -s
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.28s
     Running `target/debug/remote_server_cli restore --backup-name D2024_08_26T06_01_27 -s`
Error: "UNKNOWN" ("\"\\\"database \\\\\\\"omsupply\\\\\\\" is being accessed by other users\\\"\"")
```

Reply:
I agree ! I also noticed and added to PR description (to relocate to new issue at some point)